x-pack/plugin/ilm/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/indexlifecycle/CCRIndexLifecycleIT.java

```diff
@@ -108,6 +108,15 @@ public void testBasicCCRAndILMIntegration() throws Exception {
         }
     }
 
+    public void test() throws IOException {
```

Comment:
This doesn't appear to be used anywhere and should be removed.

If I'm wrong and it is used, it should have a more descriptive name.